src/Blocks/components/admin-settings-section/admin-settings-section-admin.scss

```diff
@@ -102,13 +102,13 @@
 		position: relative;
 		z-index: 2;
 		display: flex;
-		flex-direction: column;
+		flex-direction: row;
 		align-items: center;
 		font-size: 14px;
 	}
 
 	&__menu-link {
-		padding: 13px 5px;
+		padding: 13px 15px;
```

Comment:
forms don't use `calc` for sizes?

Reply:
noup this is admin part so there is no need